contracts/interfaces/masterchefjoev2.sol

```diff
@@ -27,6 +27,68 @@ interface IMasterChefJoeV2 {
     function poolLength() external view returns (uint256);
 
 
+    function pendingTokens(uint256 _pid, address _user)
```

Comment:
According to what I read in the NatSpec docs, all interface functions need headers